dowhy/timeseries/tests/graph_test.py

```diff
@@ -0,0 +1,67 @@
+import unittest
```

Comment:
the test file should be added outside of the dowhy folder. [root]/tests/timeseries/